### DISCUSSION TITLE
Unable to load data: Request has been terminated Possible

### DISCUSSION THREAD

**Discussion by csyyy106** [argoproj/argo-cd#18002] [General]
![企业微信截图_17139545554102](https://github.com/argoproj/argo-cd/assets/112543052/d04a4ec8-4510-45d4-84da-18ef1213d006)
![image](https://github.com/argoproj/argo-cd/assets/112543052/06f9e7d4-8d83-4e7e-8c19-5c64a88ffa92)
![image](https://github.com/argoproj/argo-cd/assets/112543052/5f790f01-b20a-42a1-ba2d-26ad168d365c)
Does anyone know why that is? Installed several times, there has been this situation, feel the network connection is very unstable

**Comment by lknite** [comment]
That looks like your login has timedout, try logging in as 'admin', or are you running as 'admin'?

You could try downloading a utility called 'kubetail' or something similar, to let you watch all the logs at the same time, maybe there will be something in there that might help indicate.  Doesn't look like any pods are crashing, so probably not out of cpu or memory, that's good.

Is it a new cluster?  Is the cluster in the cloud so we can probably trust it, or something that you stood up on your own possibly questionable, that we might want to talk about?  k3d?  minikube?  kubeadm using centos or redhat?

**Reply by csyyy106** [reply]
I'm using a k8s cluster deployed in ec2 with kubeadm, and I'm logged in with admin on an ubuntu machine

**Comment by csyyy106** [comment]
help
